docs/content/en/docs/concepts/cluster-topologies.md

```diff
@@ -0,0 +1,82 @@
+You can only use `eksctl` to create or delete the management cluster and the singleton standalone cluster.
+This shows examples of a mamagement cluster that deploys and manages multiple workload clusters:
+
+![Management clusters can create and manage multiple workload clusters](/images/eks-a_cluster_management.png)
+
+## What’s the difference between a management cluster and a bootstrap cluster for EKS Anywhere?
+
+A management cluster is a long-lived entity you have to actively operate.
+The _bootstrap_ cluster is a temporary, short-lived kind cluster that is created on a separate [Administrative machine]({{< relref "clusterworkflow/#administrative-machine" >}}) to facilitate the creation of an initial standalone or management cluster.
+You do not need to interact or operate the `kind` cluster.
+The `kind` cluster will be automatically deleted by the end of the initial cluster creation.
+
+## When should I deploy a management cluster?
+If you want to run three or more EKS Anywhere clusters, we recommend them to choose a management/workload cluster deployment topology because of the advantages listed in the table below.
+The EKS Anywhere Curated Packages feature recommends deploying certain packages such as the container registry package or monitoring packages on the management cluster to avoid circular dependency. 
+
+
+|        | Standalone cluster topology | Management/workload cluster topology  |
+|--------|-----------------------------|---------------------------------------|
+| **Pros**   | Save hardware resources   | Isolation of secrets                |
+|        | Reduced operational overhead of maintaining a separate management cluster | Resource isolation between different teams. Reduced noisy-neighbor effect. |
+|        |                             |  Isolation between development and production workloads. |
+|        |                             |  Isolation between applications and fleet management services, such as monitoring server or container registry. |
+|        |                             |  Provides a central control plane and API to automate cluster lifecycles |
+| **Cons** |  Shared secrets such, as SSH credentials or VMware credentials, across all teams who share the cluster. |  Consumes extra resources. |
+|        |  Without a central control plane (such as a parent management cluster), it is not possible to automate cluster creation/deletion with advanced methods like GitOps or IaC. |The creation/deletion of the management cluster itself can’t be automated through GitOps or IaC. |
+|        | Circular dependencies arise if the cluster has to host a monitoring server or a local container registry. | 
+||||
+
+
+## Which EKS Anywhere features support the management/workload cluster deployment topology today?
+
+|        | Supported | Notes  |
```

Comment:
Remote the "Notes" column. We should not publish feature ETAs

Reply:
Done.